docs/plugins/mailchimp.rst

```diff
@@ -3,4 +3,47 @@
 MailChimp
 #########
 
-.. vale on
\ No newline at end of file
+.. vale on
+This plugin can send contacts to MailChimp cilents based on Contact actions or Point Triggers.
+
+## Authorize
```

Comment:
```suggestion
Authorize
*********
```

Please can you fix the headings here, so they use the underline style rather than Markdown?

Here's the syntax to use:

```
H1 ###
H2 ***
H3 ===
H4 ---
H5 ~~~
```

The underline must be the same length as the heading, and your headings must be unique (as they are used to auto-generate links)